integration/nri_test.go

```diff
@@ -456,7 +457,7 @@ func TestNriLinuxCpusetAdjustmentUpdate(t *testing.T) {
 	chk, err := waitForFileAndRead(filepath.Join(out, "result"), time.Second)
 	require.NoError(t, err, "read result")
 
-	expected := "Cpus_allowed_list:\t" + availableCpuset[1] + "\n"
+	expected := "Cpus_allowed_list:\t" + availableCpuset[0] + "\n"
```

Comment:
should update[0].SetLinuxCPUSetCPUs(availableCpuset[1])  on the second container not set the first to availableCpuset[1]

Reply:
https://github.com/containerd/containerd/actions/runs/7564519481/job/20598826050?pr=9470

<img width="966" alt="image" src="https://github.com/containerd/containerd/assets/15009201/d9be6941-2e54-4a9e-8d9e-ab05cd1c8ad4">

`TestNriLinuxCpusetAdjustmentUpdate ` this test run failed, the cpus it set was 0.

Reply:
nod only failing on 1 of 8 of the matrix variations...

Reply:
yes, and every time failing job is [[Linux Integration (io.containerd.runc.v2, crun, actuated-arm64-4cpu-16gb, systemd)](https://github.com/containerd/containerd/actions/runs/7580492540/job/20646453104?pr=9470#logs)] this.